README.md

```diff
@@ -145,6 +145,7 @@ in February 2016.
 |[Trier][trier-wikipedia]|[City of Trier][trier-markets]|
 |[Tübingen][tübingen-wikipedia]|[Wochenmarkt - Universitätsstadt Tübingen][tübingen-markets]|
 |[Ulm][ulm-wikipedia]|[City of Ulm][ulm-markets]|
+|[Usingen][usingen-wikipedia]|[City of Usingen][usingen-markets]|
```

Comment:
You are missing the links here as you can see when you scroll down the README here: https://github.com/telegnom/wo-ist-markt.github.io